### PR TITLE
Merge generated BOM with services BOM

### LINKED ISSUE #2125: Include Snyk and OSV services.bom.json
I think SNYK and OSV should be added as inbound service in `services.bom.json`?

### PR DESCRIPTION
**Pull request by nscuro**
### Description

This PR adds an execution of the CycloneDX CLI to the build in order to merge the generated `bom.json` with the manually-maintained `services.bom.json`.

The services OSV and Snyk have been added to `services.bom.json` in order to reflect our new integrations.

### Addressed Issue

Relates to #1559
Closes #2125 

### Additional Details

As the merge step requires the CycloneDX CLI to be installed, it is disabled by default.

In order to keep things simple, generation and distribution of the BOM in XML format is disabled.
Focusing on just the JSON BOM should suffice. Users who need a different format can convert on their end.

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/github-templates/CONTRIBUTING.md#pull-requests)
- [ ] ~This PR fixes a defect, and I have provided tests to verify that the fix is effective~
- [ ] ~This PR implements an enhancement, and I have provided tests to verify that it works as intended~
- [ ] ~This PR introduces changes to the database model, and I have added corresponding [update logic](https://github.com/DependencyTrack/dependency-track/tree/master/src/main/java/org/dependencytrack/upgrade)~
- [ ] ~This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly~

### CODE REVIEW

File: src/main/resources/services.bom.json [hunk 1]
```diff
@@ -416,6 +416,29 @@
         }
       ]
     },
+    {
+      "bom-ref": "service:dev.osv",
+      "provider": {
+        "name": "Google",
```
Comment (VinodAnandan) [inline src/main/resources/services.bom.json:422]: Google OSV

File: src/main/resources/services.bom.json [hunk 2]
```diff
@@ -416,6 +416,29 @@
...
+        "url": [
+          "https://google.com"
```
Comment (VinodAnandan) [inline src/main/resources/services.bom.json:424]: https://osv.dev/

**Comment by nscuro** [comment]
@stevespringett Does this work for you? In particular us not publishing our BOM in XML format anymore.

**Review by stevespringett — APPROVED** [review]
LGTM. Thanks @nscuro

**Pull request merged**